qiskit/qpy/binary_io/circuits.py

```diff
@@ -755,13 +755,12 @@ def _write_instruction(
         or isinstance(instruction.operation, library.BlueprintCircuit)
     ):
         gate_class_name = instruction.operation.name
-        if version >= 11:
-            # Assign a uuid to each instance of a custom operation
-            gate_class_name = f"{gate_class_name}_{uuid.uuid4().hex}"
+        # Assign a uuid to each instance of a custom operation
+        gate_class_name = f"{gate_class_name}_{uuid.uuid4().hex}"
         # ucr*_dg gates can have different numbers of parameters,
         # the uuid is appended to avoid storing a single definition
         # in circuits with multiple ucr*_dg gates.
-        elif instruction.operation.name in {"ucrx_dg", "ucry_dg", "ucrz_dg"}:
+        if instruction.operation.name in {"ucrx_dg", "ucry_dg", "ucrz_dg"}:
             gate_class_name = f"{gate_class_name}_{uuid.uuid4()}"
```

Comment:
I think we're duplicating the uuid append to the ucr* gates now?

Reply:
Ah good catch, I was just mechanically updating the logic to remove the version branch.

Reply:
Fixed in: https://github.com/Qiskit/qiskit/pull/13917/commits/36c2623d8abb98bec5b347668d2f0aa92748cad9